### PR TITLE
fix: broken bom tree view and remove duplicate button

### LINKED ISSUE #28498: BOM Tree Broken
## Description of the issue

When you open BOM in tree view , the UI appears broken 

## Screenshot 

image

## Context information 

ERPNext: v13.14.1 (version-13)
Frappe Framework: v13.14.1 (version-13)

### Observed result

The BOM Tree view appears broken 

### Expected result

The BOM Tree view should appear appropriately

### PR DESCRIPTION
**Pull request by bhavesh95863**
Tree view looks broken during render so removed duplicate button. Also make item button visible.
Before
image

After
image

### CODE REVIEW

**Comment by ankush** [comment]
@mergifyio backport develop

File: erpnext/manufacturing/doctype/bom/bom_item_preview.html [hunk 1]
```diff
@@ -16,26 +16,15 @@ <h4>
 			</div>
 			<hr style="margin: 15px -15px;">
 			<p>
-				{% if data.value %}
+				{% if data.value != "BOM" %}
```
Comment (ankush) [inline erpnext/manufacturing/doctype/bom/bom_item_preview.html]: This is now showing button for child item without BOM too. It should keep previous check and add this new one IMO. 
```suggestion
				{% if data.value && data.value != "BOM" %}
```

File: erpnext/manufacturing/doctype/bom/bom_item_preview.html [hunk 2]
```diff
@@ -16,26 +16,15 @@ <h4>
...
 				<a style="margin-right: 7px; margin-bottom: 7px" class="btn btn-default btn-xs" href="#Form/BOM/{{ data.value }}">
```
Comment (ankush) [inline erpnext/manufacturing/doctype/bom/bom_item_preview.html]:
```suggestion
				<a style="margin-right: 7px; margin-bottom: 7px" class="btn btn-default btn-xs" href="/app/bom/{{ data.value }}">
```

File: erpnext/manufacturing/doctype/bom/bom_item_preview.html [hunk 3]
```diff
@@ -16,26 +16,15 @@ <h4>
...
 					{{ __("Open BOM {0}", [data.value.bold()]) }}</a>
 				{% endif %}
 				{% if data.item_code %}
-				<a class="btn btn-default btn-xs" href="#Form/Item/{{ data.item_code }}">
+				<a style="margin-right: 7px; margin-bottom: 7px"  class="btn btn-default btn-xs" href="#Form/Item/{{ data.item_code }}">
```
Comment (ankush) [inline erpnext/manufacturing/doctype/bom/bom_item_preview.html]:
```suggestion
				<a style="margin-right: 7px; margin-bottom: 7px"  class="btn btn-default btn-xs" href="/app/item/{{ data.item_code }}">
```

**Review by ankush — CHANGES_REQUESTED** [review]
Minor improvements, else LGTM.

**Comment by bhavesh95863** [comment]
thank @ankush . Let me know if i need to anything to merge this PR.

**Comment by ankush** [comment]
@bhavesh95863 check suggested changes. You can commit them from github too if you agree with them. First suggestion is kinda mandatory because otherwise behaviour is buggy.

**Review by ankush — APPROVED** [review]

**Pull request merged**